### ISSUE TITLE
Launching Games on Steam Deck in Gaming mode are stuck on infinite loop of Steam Loading Screen.

### ISSUE THREAD

**Issue by resolutezach1** [lutris/lutris#5792]
### Bug description

I'm currently using the latest flatpak version of Lutris on my Steam Deck and all my games are broken since they won't show any video in Gaming (but they still show in Desktop Mode)

### How to Reproduce

1. Launch Any Lutris Game added to Steam in Gaming Mode

### Expected behavior

Each Lutris Game Launched in Steam Deck Gaming Mode should show the game accordingly without being stuck in Steam's Loading Screen.

### Log output

```shell
[Left Blank since I'm not sure it would make a difference]
```

### System Information

```shell
Vulkan support: YES
Esync support: YES
Fsync support: YES
Wine installed: YES
Gamescope: NO
Mangohud: NO
Gamemode: YES
Steam: YES
In Flatpak: YES
[System]
OS: GNOME 46 Flatpak runtime
Arch: x86_64
Kernel: 6.5.0-valve22-1-neptune-65-g9a338ed8a75e
Lutris Version: 0.5.18
Desktop: KDE
Display Server: x11
[CPU]
Vendor: AuthenticAMD
Model: AMD Custom APU 0405
Physical cores: 4
Logical cores: 8
[Memory]
RAM: 14.5 GB
Swap: 8.2 GB
[Graphics]
Vendor: AMD
OpenGL Renderer: AMD Custom GPU 0405 (radeonsi, vangogh, LLVM 17.0.6, DRM 3.57, 6.5.0-valve22-1-neptune-65-g9a338ed8a75e)
OpenGL Version: 4.6 (Compatibility Profile) Mesa 24.2.5 (git-3b9fcb7e4d)
OpenGL Core: 4.6 (Core Profile) Mesa 24.2.5 (git-3b9fcb7e4d)
OpenGL ES: OpenGL ES 3.2 Mesa 24.2.5 (git-3b9fcb7e4d)
Vulkan Version: 1.3.261
Vulkan Drivers: AMD Custom GPU 0405 (RADV VANGOGH) (1.3.289)
```

**Comment by AlmeidaRW** [comment]
Hi, I have the same issue, but only when using GE-Proton as a runner. Do you have an issue if you set the runner to wine-ge-8-26?

**Comment by resolutezach1** [comment]
Yeah, I did. But then I Figured I'd re-install all my Lutris Games, and that actually fixed it. I'm guessing it's a problem with the wine prefixes, I could be wrong.

**Comment by AlmeidaRW** [comment]
> Yeah, I did. But then I Figured I'd re-install all my Lutris Games, and that actually fixed it. I'm guessing it's a problem with the wine prefixes, I could be wrong.

Sorry, I don't follow. Is it working in Gaming mode now with ge-proton or wine-ge? The default is wine-ge, so unless you specifically set it to ge-proton, you're still using wine-ge, which has always worked

**Comment by resolutezach1** [comment]
Wine-ge had problems too before I re-installed my games, but I haven't tried ge-proton yet.

**Comment by gh0stsp3ctre** [comment]
Getting this issue also. Regardless of what runner I use. Even experimental and this only happens when in game mode.

**Comment by 9E307** [comment]
This also happens to me in game mode. It is true that only the default wine-ge-8-26 can work properly. Other versions cannot play the game normally. At the same time, any wine version can run the game normally in desktop mode without any problem.

**Comment by matty-r** [comment]
This may be related to #5509 I had a similar issue but noticed I could hear the game in the background. Adding `PROTON_VERB=waitforexitandrun` in Lutris as an environment variable is a workaround in the meantime. But sometimes the steam overlay won't show correctly when using this.

**Comment by 9E307** [comment]
One thing worth noting is that after my testing, when the game is installed via GOG, there will be no such problem. Any version of wine can run the game normally in game mode without the infinite loop of the Steam loading screen. Therefore, this situation only occurs on the Epic platform.

**Comment by szombi** [comment]
> This may be related to [#5509](https://github.com/lutris/lutris/issues/5509) I had a similar issue but noticed I could hear the game in the background. Adding `PROTON_VERB=waitforexitandrun` in Lutris as an environment variable is a workaround in the meantime. But sometimes the steam overlay won't show correctly when using this.


Yeah, same here. Yesterday, I finally could play games installed from the Epic Game Store, but when I quit, the Steam overlay wouldn't work properly, and I couldn't close the Epic Store itself.

**Comment by ayasme** [comment]
Happening to me as well. Re-installed Lutris flatpak, made a fresh prefix with wine-ge-8-26, and getting the infinite loading screen in game mode.

**Comment by R1kaB3rN** [comment]
> But sometimes the steam overlay won't show correctly when using this.

I’d be interested in looking into the Steam overlay issue if you or anyone else here can describe the problem in more detail and list a simple way to reproduce it.

**Comment by CaffeinatedDragonWare** [comment]
I'm have the same issue. If I close any lutris game either using steam or through the game itself, the ui freezes. I tried both the new version of lutris and the version release last year but its the same issue. Not a problem in desktop mode, except if you try to launch the through big picture (not game mode) it will freeze the same way. I'm willing to work with you on this.

**Comment by legluondunet** [comment]
Hello , is this issue still present in latest Lutris version 0.5.22?
Without answer, this ticket will be closed in 7 days.

**Comment by CutMyFace** [comment]
Yes it is. I'm on Steam Deck running Bazzite, and it seems like it's closing itself to open the game in the background — the play button in Steam desktop mode is turning green, and the game opens up without Steam overlay. In the game mode it just shows loading screen, the game itself is running in the background — I can still hear it.
